### PR TITLE
PCM-2807 strataService in common missing articles.get

### PR DESCRIPTION
**Pull request by engineersamuel**
Adding articles.get so we can fetch the full html content for articles, PCM-2807

@vrathee @gandhikeyur @jtrantin  please review and push the new release to bower.

### CODE REVIEW

File: app/common/services/strataService.js [hunk 1]
```diff
@@ -197,6 +197,25 @@ angular.module('RedhatAccess.common').factory('strataService', [
                     return deferred.promise;
                 }
             },
+            articles: {
+                get: function (uri) {
+                    var deferred = $q.defer();
+                    var splitUri = uri.split('/');
+                    uri = splitUri[splitUri.length - 1];
```
Comment (jtrantin) [inline app/common/services/strataService.js]: Correct me if I am wrong, but this code seems to parse the article ID out of the URI. Why are you not passing the ID of the article instead? Seems like a hassle to pass the URI just to extract the ID out of it.

Comment (engineersamuel) [inline app/common/services/strataService.js]: I literally copy and pasted from the service part fetching solutions.  We could definitely rewrite it to be more sane but I want going to spend too much time on it in hopes that we can get this functionality in uds.  I have a Jira open for getting the HTML in solutions and articles.  Feel free to reject this though and I'll rewrite it on Monday.

Comment (engineersamuel) [inline app/common/services/strataService.js]: Oh and you can just pass the id, you don't have to pass the full uri

Comment (jtrantin) [inline app/common/services/strataService.js]: Still, seems rather hackish for no good reason. I personally would rewrite this, but let's wait for opinions from others :).

File: app/common/services/strataService.js [hunk 2]
```diff
@@ -197,6 +197,25 @@ angular.module('RedhatAccess.common').factory('strataService', [
...
+                    if (!ie8 && strataCache.get('article' + uri)) {
```
Comment (vrathee) [inline app/common/services/strataService.js]: Since we don't support ie8 now.. we can get rid of it.

**Comment by vrathee** [comment]
Apart from what @jtrantin said regarding using actual article ID if we have it and the ie8 part. Rest Seems good.

**Comment by renujhamtani** [comment]
@engineersamuel  There are comments from developers. Are we solving this or should i merge this?

**Comment by engineersamuel** [comment]
@renujhamtani not quite yet, I'm about to push a few changes, then we can merge.

**Comment by engineersamuel** [comment]
@jtrantin ok, cleaned up the code and rewrote it not following the prior way.  Ultimately we probably need to rewrite all methods to clean them up.  That'll be for a later time though.

@renujhamtani we should be good to merge now.

**Comment by jtrantin** [comment]
Looks great :)

**Comment by vrathee** [comment]
Reviewed!

**Pull request merged**